frontend/src/pages/org/index.ts

```diff
@@ -122,6 +124,8 @@ export class Org extends BtrixElement {
   @state()
   private isCreateDialogVisible = false;
 
+  private readonly [searchOrgContextKey] = new SearchOrgContextController(this);
```

Comment:
Small thing, but is there any particular reason to use a symbol as a key here? It's only used in this one page anyways, so I don't think it matters much either way, but it doesn't seem necessary to me.

Reply:
Using a symbol avoids thinking of a context key string that's completely unique in the entire app. It's unlikely that we'd incorporate a third-party library that uses a context named `search-values` but I think it's good practice to establish regardless. Otherwise, we could follow a convention like prefixing the context name with `btrix-`?